tests/_support/Step/Acceptance/Reports.php

```diff
@@ -51,6 +51,8 @@ public function createAccount($name)
         $Sidebar->clickSideBarAction('Create');
         $I->waitForEditViewVisible();
         $I->fillField('#name', $name);
+        $I->fillField('#phone_office', '(810) 267-0146');
+        $I->fillField('#website', 'www.afakeurl.com');
```

Comment:
better to use www.example.com/ in examples!

Reply:
:+1: will replace with fzaninotto/faker for generating fake data.